pkg/diskmaker/diskmaker.go

```diff
@@ -124,10 +124,22 @@ func (d *DiskMaker) Run(stop <-chan struct{}) {
 	}
 }
 
-func (d *DiskMaker) createSymLink(deviceNameLocation DiskLocation, symLinkPath string, symLinkDirPath string, baseDeviceName string) {
+func (d *DiskMaker) createSymLink(deviceNameLocation DiskLocation, symLinkDirPath string) {
+
+	var symLinkSource, symLinkTarget string
+	var isSymLinkedByDeviceName bool
+	if deviceNameLocation.diskID != "" {
+		symLinkSource = deviceNameLocation.diskID
+		symLinkTarget = getSymlinkTarget(deviceNameLocation.diskID, symLinkDirPath)
+	} else {
+		symLinkSource = deviceNameLocation.diskNamePath
+		symLinkTarget = getSymlinkTarget(deviceNameLocation.diskNamePath, symLinkDirPath)
+		isSymLinkedByDeviceName = true
+	}
+
```

Comment:
Can we add some tests for this code?

Reply:
added unit tests for `createSymLink` method.